src/app/app.component.ts

```diff
@@ -51,6 +51,7 @@ export class MyApp {
       let isIE: boolean = navigator.userAgent.indexOf('Trident', 0) !== -1;
       this.infoSvc.setInternetExplorer(isIE);
       this.runningInBrowser = this.platform.is('mobileweb') || this.platform.is('core');
+      this.notOnDesktop = !this.platform.is('core');
```

Comment:
Why the double negative? Maybe it's just me, but we could call it onDesktop and no longer need the `!`.